test-network-function/identifiers/identifiers.go

```diff
@@ -321,15 +326,24 @@ they are the same.`),
 	TestICMPv4ConnectivityIdentifier: {
 		Identifier: TestICMPv4ConnectivityIdentifier,
 		Type:       normativeResult,
-		Remediation: `Ensure that the CNF is able to communicate via the Default OpenShift network.  In some rare cases,
-CNFs may require routing table changes in order to communicate over the Default network.  In other cases, if the
-Container base image does not provide the "ip" or "ping" binaries, this test may not be applicable.  For instructions on
-how to exclude a particular container from ICMPv4 connectivity tests, consult:
-[README.md](https://github.com/test-network-function/test-network-function#issue-161-some-containers-under-test-do-not-contain-ping-or-ip-binary-utilities).`,
+		Remediation: `Ensure that the CNF is able to communicate via the Default OpenShift network. In some rare cases,
+CNFs may require routing table changes in order to communicate over the Default network. To exclude a particular container
+from ICMPv4 connectivity tests, add the test-network-function.com/skip_connectivity_tests label to it. The label value is not important, only its presence.`,
 		Description: formDescription(TestICMPv4ConnectivityIdentifier,
 			`checks that each CNF Container is able to communicate via ICMPv4 on the Default OpenShift network.  This
-test case requires the Deployment of the debug daemonset.
-`),
+test case requires the Deployment of the debug daemonset.`),
+		BestPracticeReference: bestPracticeDocV1dot2URL + " Section 6.2",
+	},
+
+	TestICMPv4ConnectivityIdentifierMultus: {
+		Identifier: TestICMPv4ConnectivityIdentifierMultus,
+		Type:       normativeResult,
+		Remediation: `Ensure that the CNF is able to communicate via the Multus network(s). In some rare cases,
+CNFs may require routing table changes in order to communicate over the Multus network(s). To exclude a particular container
```

Comment:
Label is on the pod. We used to support skipping specific container with static config. That got dropped at some point.

Reply:
Fixed, thanks!